packages/cra-template-typescript/template/src/App.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import logo from './logo.svg';
 import './App.css';
 
-const App: React.FC = () => {
+const App = () => {
```

Comment:
@iansu, @mrmckeb thoughts on being more direct here to point people at our documentation?

```suggestion
// Learn more about TypeScript and React at
// https://create-react-app.dev/docs/adding-typescript/#getting-started-with-typescript-and-react
const App = () => {
```